js/src/crypto.ts

```diff
@@ -156,6 +179,132 @@ export class Crypto {
       throw new Error('Signature verification failed: Integrity mismatch')
     }
   }
+
+  /**
+   * signs the message as a compact jws with nonce
+   * @param privateKeyJwk - the key to sign with
+   * @param kid - the kid to include in the jws header. used by the verifier to select the appropriate verificationMethod
+   *              when dereferencing the signer's DID
+   * @returns the compact JWS
+   */
+  static async token(opts: TokenOptions): Promise<string> {
```

Comment:
what is the difference between this method and the `sign` method in line 85?

Reply:
This is for creating the bearer token for get requests and sign populates the `signature` field on a tbdex message

Reply:
ah, i see they're slightly different in implementation - sign() uses `Crypto.hash(jwspayload)` while token() does not. cool!

Reply:
i was originally thinking that they should be squished into one method, but that part is distinct

Reply:
eventually i wanna refactor to reduce duplication